boot/bootdata/txtsetup.sif

```diff
@@ -224,13 +224,15 @@ pci_up = "Standard PC Uniprocessor"
 ;pci_mp = "Standard PC Multiprocessor"
 acpi_up = "ACPI PC Uniprocessor"
 ;acpi_mp = "ACPI PC Multiprocessor"
+xbox = "Original Xbox"
 
 [Map.Computer]
 ;<id> = <pnp id string>
 pci_up = "PC UP"
 ;pci_mp = "PC MP"
 acpi_up = "ACPI UP"
 ;acpi_mp = "ACPI MP"
+xbox = "Xbox"
```

Comment:
Also consider "Original Xbox"

Reply:
This one is some internal string, it doesn't display in setup or in device manager